### PR TITLE
e2e - form definition submission configuration test

### PR DESCRIPTION
**Pull request by howard-zhang**
- form definition submission configuration test
- some updates on existing test steps for issues found during testing

### CODE REVIEW

File: apps/tenant-management-webapp-e2e/src/integration/form.feature [hunk 1]
```diff
@@ -75,7 +75,97 @@ Feature: Form
...
+    When the user clicks "arrow up" for the disposition state of "Documents needed", "Need to supply required documents"
+    Then the user views the disposition state of "Documents needed", "Need to supply required documents" being row "1"
+    When the user clicks "arrow down" for the disposition state of "Approved", "The application is approved"
+    Then the user views the disposition state of "Approved", "The application is approved" being row "3"
+    # Edit states
+    When the user clicks Save button on form definition editor
+    And the user clicks Back button on form definition editor
+    And the user clicks "Edit" button for the form definition of "autotest-form-disposition-states", "DO NOT DELETE"
+    When the user clicks "Submission configuration" tab in form definition editor
+    When the user clicks "edit" button for the disposition state of "Approved", "The application is approved"
```
Comment (LanaDIO) [inline apps/tenant-management-webapp-e2e/src/integration/form.feature:134]: When should be Then or And

File: apps/tenant-management-webapp-e2e/src/integration/form.feature [hunk 2]
```diff
@@ -75,7 +75,97 @@ Feature: Form
...
+    Then the user views Edit disposition state modal
+    When the user enters "Reviewed" as name and "The application is reviewed" as description in Edit disposition state modal
+    And the user clicks Save button in disposition state modal
+    Then the user "views" the disposition state of "Reviewed", "The application is reviewed"
+    # # Delete states
+    When the user clicks Save button on form definition editor
+    And the user clicks Back button on form definition editor
+    And the user clicks "Edit" button for the form definition of "autotest-form-disposition-states", "DO NOT DELETE"
+    When the user clicks "Submission configuration" tab in form definition editor
+    When the user clicks "Delete" button for the disposition state of "Reviewed", "The application is reviewed"
```
Comment (LanaDIO) [inline apps/tenant-management-webapp-e2e/src/integration/form.feature:149]: When should be Then or And

File: apps/tenant-management-webapp-e2e/src/integration/form.feature [hunk 3]
```diff
@@ -75,7 +75,97 @@ Feature: Form
...
+    When the user clicks "Delete" button for the disposition state of "Rejected", "The application is rejected"
+    Then the user views delete "disposition state" confirmation modal for "Rejected"
+    When the user clicks Delete button in delete confirmation modal
+    Then the user "should not view" the disposition state of "Rejected", "The application is rejected"
+    When the user clicks Save button on form definition editor
+    And the user clicks Back button on form definition editor
+    And the user clicks "Edit" button for the form definition of "autotest-form-disposition-states", "DO NOT DELETE"
+    When the user clicks "Submission configuration" tab in form definition editor
+    Then the user "should not view" the disposition state of "Reviewed", "The application is reviewed"
+    Then the user "should not view" the disposition state of "Documents needed", "Need to supply required documents"
```
Comment (LanaDIO) [inline apps/tenant-management-webapp-e2e/src/integration/form.feature:170]: there is 3 Then

File: apps/tenant-management-webapp-e2e/src/integration/form.feature (hunk 1 above)
Comment (howard-zhang) [reply apps/tenant-management-webapp-e2e/src/integration/form.feature:134]: Updated to And

File: apps/tenant-management-webapp-e2e/src/integration/form.feature (hunk 3 above)
Comment (howard-zhang) [reply apps/tenant-management-webapp-e2e/src/integration/form.feature:170]: Updated to use proper keywords

File: apps/tenant-management-webapp-e2e/src/integration/form.feature (hunk 2 above)
Comment (howard-zhang) [reply apps/tenant-management-webapp-e2e/src/integration/form.feature:149]: Updated to Add

**Pull request merged**